src/mixins/namex-request-mixin.ts

```diff
@@ -62,15 +63,16 @@ export default class NamexRequestMixin extends Mixins(DateMixin) {
   }
 
   /**
-   * Generates name request state for the store.
+   * Parses name request.
    * @param nr the name request response payload
    * @param filingId the filing id
    */
-  generateNameRequestState (nr: any, filingId: number): NameRequestIF {
+  parseNameRequest (nr: any, filingId: number): NameRequestIF {
     const approvedName = nr.names.find(name => name.state === NameRequestStates.APPROVED).name
     return {
-      nrNumber: nr.nrNum,
-      // TODO: Update entityType to use nr.requestTypeCd when namex supports our entity types
+      // workaround for old or new property name
+      nrNumber: nr.nrNum || nr.nrNumber,
```

Comment:
Sorry, whats the difference between nrNum & nrNumber?

Reply:
"ber"

nrNum is what the old property is called in the namex NR. nrNumber is what we've been calling it everywhere else in auth-web, filings UI, create UI and Legal API. So I added this code to support both the old name and the new name. If/when namex changes to nrNumber, then we can remove our workaround (it's in a few places, always with the same comment to make it easier to find).